docs/apache-airflow-providers-databricks/operators/workflow.rst

```diff
@@ -0,0 +1,47 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/operator:DatabricksWorkflowTaskGroup:
+
+
+DatabricksWorkflowTaskGroup
+===========================
+
+Use the :class:`~airflow.providers.databricks.operators.databricks_workflow.DatabricksWorkflowTaskGroup` to launch and monitor
+notebook job runs on Databricks as Airflow tasks.
+
+
+
+Examples
+--------
+
+Example of what a DAG looks like with a DatabricksWorkflowTaskGroup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. exampleinclude:: /../../tests/system/providers/databricks/example_databricks_workflow.py
+    :language: python
+    :start-after: [START howto_databricks_workflow_notebook]
+    :end-before: [END howto_databricks_workflow_notebook]
+
```

Comment:
It would be great if the docs could include a screenshot of how the DAG is rendered in Airflow and how it launches the Databricks Workflow in Databricks.

Reply:
added a screeshot. this is how the docs page appears now

![screencapture-localhost-63342-airflow-docs-build-docs-apache-airflow-providers-databricks-stable-operators-workflow-html-2024-05-29-22_10_49](https://github.com/apache/airflow/assets/10206082/9ff5c648-8252-4cbb-a0db-7ca500665f58)